src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
-body {
-  margin: 0;
+$color-bg-white: #fff;
+$color-text-dark: #060b35;
+$color-text-gray: #616070;
+$main-color: #00acdc;
+
+.page {
+  font-family: Roboto, Aria, sans-serif;
```

Comment:
There is a typo in the font-family declaration. It should be 'Arial' instead of 'Aria'.